src/instruction.rs

```diff
@@ -94,3 +98,14 @@ pub fn noop() -> Instruction {
     data: PythClientInstruction::Noop.try_to_vec().unwrap(),
   }
 }
+
+// Returns ok if price is not stale
+pub fn price_not_stale(price_account_data: Vec<u8>) -> Instruction {
```

Comment:
this would be more flexible if the instruction took an argument that was the expected status. E.g.,  `test_status(account_data, expected_status)`

Reply:
also why `Vec<u8>` instead of the struct? If there's a good reason, i recommend leaving a comment

Reply:
Yes both makes sense. it was Vec<u8> because previously logic was embedded into load_price. I'll fix it.

Reply:
Added a comment to explain it.